Tests/MapboxMapsTests/Gestures/GestureHandlers/DoubleTapToZoomInGestureHandlerTests.swift

```diff
@@ -61,4 +61,16 @@ final class DoubleTapToZoomInGestureHandlerTests: XCTestCase {
         easeToCompletion(.end)
         XCTAssertEqual(delegate.animationEndedStub.parameters, [.doubleTapToZoomIn])
     }
+
+    func testFocalPoint() {
+        let focalPoint = CGPoint(x: 1000, y: 1000)
+        gestureHandler.focalPoint = focalPoint
+        gestureRecognizer.getStateStub.defaultReturnValue = .recognized
+        mapboxMap.cameraState = .random()
+
+        gestureRecognizer.sendActions()
+
+        XCTAssertEqual(cameraAnimationsManager.easeToStub.invocations.count, 1)
+        XCTAssertEqual(cameraAnimationsManager.easeToStub.parameters[0].camera.anchor, focalPoint)
```

Comment:
```suggestion
        XCTAssertEqual(cameraAnimationsManager.easeToStub.parameters.first?.camera.anchor, focalPoint)
```
Avoid crashing the tests if easeTo isn't invoked.

Reply:
There are several other places in the PR that access array elements without checking the array length. I'm only going to comment on this one, but let's address all of them.

Reply:
Good point, fixed it.